### PR TITLE
Added Equitel network

### PR DESCRIPTION
**Pull request by kamikazechaser**
Added Equitel network

Source: http://www.equitel.com/my-money/rates

### CODE REVIEW

File: data/equitel.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
+{
+  "name": "equitel",
+  "meta": {
+    "spec": "0.1",
```
Comment (GochoMugo) [inline data/equitel.json]: The spec version should corresponding to the one you are using in this data file. This should most probably be `0.4` (the latest version)!

**Comment by kamikazechaser** [comment]
```Try to be precise in the descriptions. E.g. paybills is kinda ambiguous!``` 

Thats the essence of Equitel! **all bills supported on its platform** do not incur any charge!

image 001 19 56

File: data/debug.log [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[1111/191719:ERROR:crash_report_database_win.cc(562)] CreateDirectory C:\Program Files (x86)\Easthas\Reports\Dump: The system cannot find the path specified. (0x3)
```
Comment (GochoMugo) [inline data/debug.log]: ?

**Comment by kamikazechaser** [comment]
Fixed!

File: data/equitel.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        {
+          "name": "equitel or orange money",
+          "ranges": [
+            { "low": "-Infinity", "high": 49, "amount": -1 },
+            { "low": 50,     "high": 100,     "amount": 0 },
+            { "low": 101,    "high": 500,    "amount": 0 },
+            { "low": 501,   "high": 1000,    "amount": 0 },
+            { "low": 1001,   "high": 1500,    "amount": 0 },
+            { "low": 1501,   "high": 35000,   "amount": 0 },
+            { "low": 70001,  "high": "+Infinity", "amount": -1 }
```
Comment (GochoMugo) [inline data/equitel.json]: `low` should be `35001`!

File: data/equitel.json [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  },
+  "transactions": [
+    {
+      "name": "transfer",
+      "classes": [
+        {
+          "name": "equitel or orange money",
+          "ranges": [
+            { "low": "-Infinity", "high": 49, "amount": -1 },
+            { "low": 50,     "high": 100,     "amount": 0 },
```
Comment (GochoMugo) [inline data/equitel.json]: you could just use one range i.e. `"low": 50, "high": 35000, "amount": 0`!

File: data/equitel.json [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    {
+      "name": "others",
+      "amount_input": false,
+      "classes": [
+        {
+          "name": "deposits",
+          "amount": 0
+        },
+        {
+          "name": "transaction reports",
```
Comment (GochoMugo) [inline data/equitel.json]: where you referring to, for this one? I cant seem to find it!

**Comment by GochoMugo** [comment]
I will have to add support for **fractional parts** in the schema, for this to work accordingly!

File: data/equitel.json (hunk 5 above)
Comment (kamikazechaser) [inline data/equitel.json]: https://www.youtube.com/watch?v=rp_YdWqg4gM

Comment (kamikazechaser) [inline data/equitel.json]: https://www.facebook.com/KeEquityBank/posts/723401547805474

**Pull request merged**